main.cpp

```diff
@@ -559,8 +731,114 @@ bool Drone::land()
 #include <iostream>
 int main()
 {
-    Example::main();
+    // Example::main();
```

Comment:
Don't change code that does not need to be changed
uncomment this line.
- [ ] revise